addons/mail/tests/discuss/test_discuss_channel.py

```diff
@@ -540,6 +540,7 @@ def test_mail_message_starred_group(self):
         message in a group should be considered only if:
             - It's our message
```

Comment:
This is now false, but I still don't understand why we change starred counter in the leave channel where we just want to avoid crashes when leaving.